src/include/partitioning/custom_part_lib.rb

```diff
@@ -293,13 +293,17 @@ def check_ok_fssize(size_k, volume)
 
       if Ops.less_than(size_k, min_size_k)
```

Comment:
@dwaas Yes. All those `Ops` and `Builtins`  methods are ugly leftovers from the autoconversion from YCP to Ruby that we will like to see dying.

They were introduced because YCP and Ruby operators had different behavior in 1% of situations, but every time we touch code we try to remove them in favor of standard Ruby. As said, in 99% of occasions it just means an straightforward substitution like the one you mentioned.